### PR TITLE
Display pending rows again

### PR DESCRIPTION
**Pull request by akalin-keybase**
This was removed by #9473, but that breaks the "Start a chat" case.

### CODE REVIEW

File: shared/profile/container.js [hunk 1]
```diff
@@ -67,7 +67,9 @@ const mapDispatchToProps = (dispatch: Dispatch, {setRouteState}: OwnProps) => ({
   onAcceptProofs: (username: string) => dispatch(TrackerGen.createFollow({localIgnore: false, username})),
   onBack: () => dispatch(navigateUp()),
   onChangeFriendshipsTab: currentFriendshipsTab => setRouteState({currentFriendshipsTab}),
-  onChat: (myUsername, username) => dispatch(createStartConversation({users: [username, myUsername]})),
+  onChat: (myUsername, username) => {
+    dispatch(createStartChat({myUsername, username}))
```
Comment (akalin-keybase) [inline shared/profile/container.js]: Likely would have to replace some other instances of `createStartConversation`

**Comment by akalin-keybase** [comment]
I started doing as you suggested, @MarcoPolo, but I started having second thoughts. The first problem is that, although, it's nice that treating 'Start a chat' as a filled-in search gives most of the behavior we want, it's still a bit weird to be put into search mode when presumably the user just wants to start a one-on-one chat.

The second problem is that a new search puts the focus on the search bar, which is annoying in the 'Start a chat' case, since presumably the user just wants to start chatting right away. Fixing this would seem to require storing some state in the global state. (sigh)

However, the alternative isn't great either. We'd have to (re)add logic to display and un-display pending conversations so that they behave as expected.

Thoughts?

**Comment by MarcoPolo** [comment]
I think it's probably fine to put the user in search mode if we focus the text input first. But maybe we should first check if there is already an existing conversation and select that first, if not enter the search mode with the text input selected. But this is probably a design question?

**Comment by akalin-keybase** [comment]
https://github.com/keybase/client/pull/10015 is a better fix, I think.

**Comment by akalin-keybase** [comment]
Now obsolete.

**Pull request closed without merging**